Lib/fontbakery/profiles/universal.py

```diff
@@ -75,6 +75,7 @@
         'com.google.fonts/check/whitespace_widths',
         'com.google.fonts/check/interpolation_issues',
         'com.google.fonts/check/math_signs_width',
+        'com.adobe.fonts/check/family/consistent_family_name',
```

Comment:
I _think_ this line needs to be added to https://github.com/googlefonts/fontbakery/blob/main/Lib/fontbakery/profiles/opentype.py instead.